app/assets/stylesheets/administrate/library/_variables.scss

```diff
@@ -0,0 +1,55 @@
+// Typography
+$base-font-family: "Lato", $helvetica;
+$heading-font-family: $base-font-family;
+
+// Font Sizes
+$base-font-size: 1em;
+
+// Line height
+$base-line-height: 1.5;
+$heading-line-height: 1.2;
+
+// Other Sizes
+$modular-scale-ratio: $minor-third;
+$base-border-radius: 3px;
+$base-spacing: $base-line-height * 1em;
+$small-spacing: $base-spacing / 2;
```

Comment:
`$base-spacing / 2` should be written without spaces around the operator: `$base-spacing/2`